frontend/scss/components/molecules/agenda-item.scss

```diff
@@ -15,56 +15,122 @@
 @import 'components/atoms/_color.scss';
 
 .#{molecule('agenda-item')} {
-
   @include txt;
   @include txt-lrg;
-  @include txt-font-accent;
   display: flex;
   align-items: center;
   text-align: left;
 
+  &-image {
+    display: none;
+
+    @media (min-width: 768px) {
+      display: block;
+      border-radius: 50%;
+      align-self: flex-start;
+      margin-right: 10px;
+      margin-left: -60px;
+    }
+
+    &.multiple {
+      position: absolute;
+
+      &:nth-child(1) {
```

Comment:
Having that many `:nth-child` selectors doesn't seem like the most intuitive solution 😅